gdcdictionary/schemas/demographic.yaml

```diff
@@ -61,11 +57,27 @@ properties:
       - string
       - "null"
 
-  gender:
+  gender: #using D4b iniitially. TODO: correct distinctions?
+    description: >
+      What is your gender identity? 
     enum:
-      - Male
-      - Female
-      - Unknown
+      - "Male"
+      - "Female"
+      - "Transgender"
+      - "Gender nonconforming"
+      - "Not reported"
+
+  ethnicity: #added given difference between race/ethnicity
```

Comment:
Since it's the same as race, instead of duplicating data, you could use a "field mapping" in the data-portal config to change the prop name from "race" to "ethnicity" in the exploration page. The downside is that the data model won't match the exploration page.

Reply:
Actually this works out well - deleted ethnicity and added race to required.